Fixed click region for buttonwidget

blootsvoets opened the pull request:
Click region for the button widget was only the content. Since the whole button widget area responds to mouseover, this is a bit counterintuitive, so I made the whole button widget area respond to clicks.

damylen commented:
Thnx! Have you checked this with a clickable legends inside the button and with the edit button active for dynamic layers?

blootsvoets commented:
Ah die clicks worden dan inderdaad niet meer herkend... Ik zal kijken of ik een andere oplossing kan bedenken.

blootsvoets commented:
Moved the click handling back to content, and adjusted the height and padding of the content so that it fills the entire div. I had to override the ng-binding and s3d classes padding for this, so it's using !important.

blootsvoets commented:
@damylen do you think the pull request is acceptable now?
